.github/workflows/codeql-analysis.yml

```diff
@@ -32,11 +32,18 @@ jobs:
     - name: Checkout repository
       uses: actions/checkout@v2
 
+    - name: Python Dependencies
+      run: |
+        sudo apt-get install -y libxml2-dev libxmlsec1-dev libxmlsec1-openssl
```

Comment:
it's good to add apt-get update first otherwise this breaks when those packages get updated